docs/rest_api_ep.rst

```diff
@@ -0,0 +1,274 @@
+REST API Dynamic Endpoint Attach Detach
+=======================================
+
+This section will go over the specifics of the REST APIs supported by F5 ACI ServiceCenter application for dynamic endpoint attach detach feature.
```

Comment:
This section contains the specifics of the REST APIs supported by the F5 ACI ServiceCenter application  for the dynamic endpoint attach detach feature.

Reply:
Resolved in docs/rest_api.rst (docs/rest_api_ep.rst  is an outdated filename)